src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -509,6 +540,25 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
     }
   }
 
+  // For renewal only
+  const onAccessorRadioBtnChange = (accessType: AccessType, userId: string) => {
```

Comment:
To verify, one of these options should be selected by default.  My hope is that the backend is returning "RENEW_ACCESS" for existing accessors!

Reply:
The renew radio button will be selected by default if the user has a renewal data access type.